src/app/containers/ArticleTimestamp/index.test.jsx

```diff
@@ -42,6 +46,24 @@ describe('ArticleTimestamp', () => {
     Date.now = originalDate;
   });
 
+  describe('snapshot tests', () => {
```

Comment:
I think you can omit this line - we don't really need to know that we're running snapshot tests

Reply:
Good shout. I've removed in [#89874f0](https://github.com/bbc/simorgh/pull/5072/commits/89874f0500e827854056638c0d529517429dcdcc)